src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/list/AdminAreasOfLawControllerTest.java

```diff
@@ -33,12 +35,7 @@
     @Test
     void shouldReturnAllAreasOfLaw() throws Exception {
 
-        final List<AreaOfLaw> mockAllAreasOfLaw = Arrays.asList(
-            new AreaOfLaw(new uk.gov.hmcts.dts.fact.entity.AreaOfLaw(100, "Divorce")),
-            new AreaOfLaw(new uk.gov.hmcts.dts.fact.entity.AreaOfLaw(200, "Adoption")),
-            new AreaOfLaw(new uk.gov.hmcts.dts.fact.entity.AreaOfLaw(300, "children")),
-            new AreaOfLaw(new uk.gov.hmcts.dts.fact.entity.AreaOfLaw(400, "Money"))
-        );
+        final List<AreaOfLaw> mockAllAreasOfLaw = getAreasOfLaw();
```

Comment:
The method getAreaOfLaw() is called a few times. So a new array is constructed every time. It would be better to define a constant class variable and use the same one every time:
private static final AREAS_OF_LAW = Arrays.asList(.......);